extensions/amp-story/1.0/_locales/es-419.js

```diff
@@ -73,19 +73,19 @@ export default {
     string: 'Tumblr',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_TWITTER]: {
-    string: 'Gorjeo',
+    string: 'Twitter',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_WHATSAPP]: {
-    string: 'Whatsapp',
+    string: 'WhatsApp',
   },
   [LocalizedStringId.AMP_STORY_SYSTEM_LAYER_SHARE_WIDGET_LABEL]: {
     string: 'Compartir',
   },
   [LocalizedStringId.AMP_STORY_WARNING_DESKTOP_SIZE_TEXT]: {
-    string: 'Ampliar su ventana para ver esta experiencia',
+    string: 'Ampliar la ventana para ver esta experiencia',
   },
   [LocalizedStringId.AMP_STORY_WARNING_EXPERIMENT_DISABLED_TEXT]: {
-    string: 'Debe habilitar el experimento amp pisos para ver este contenido',
+    string: 'Debe habilitar el experimento amp-story para ver este contenido',
```

Comment:
Note: this one is deprecated and, as far as I know, should never be displayed

Reply:
That's fine. Then maybe delete it completely?